### ISSUE TITLE
Jetpack Submenu Icons Don't Open New Window

### ISSUE THREAD

**Issue by sdixon194** [Automattic/jetpack#32601]
### Impacted plugin

Jetpack

### What

When looking at the Jetpack Submenu items that indicate a new window will be opened, they don't actually open a new window.

### How

Surfaced during internal testing: p8oabR-1jo-p2#comment-7542

Some submenu items like Subscribers and Scan don't open the window in a new tab. I believe we're using this indicator because the link goes outside of the WP Site (to cloud.jetpack.com in the case of Scan), but it opens in the same time. 

![Screenshot 2023-08-21 at 10 56 30 AM](https://github.com/Automattic/jetpack/assets/33553323/30932d5d-97dc-40eb-9e8e-550190c5ab58)

**Comment by danjjohnson** [comment]
This seems related enough to add here, but let me know if a new issue is preferred.

Same thing happens if you go to My Jetpack, and click on Manage:

![image](https://github.com/Automattic/jetpack/assets/1990037/b4768857-6248-4b50-abc4-915c806d0868)

The transfer ownership link has a 'new window' icon, but it opens in the same window.

![image](https://github.com/Automattic/jetpack/assets/1990037/a53076fc-bde4-4ccf-badf-0877c139f80e)

Ref: p1698052981757229-slack-CDLH4C1UZ

**Comment by keoshi** [comment]
Related: https://github.com/Automattic/jetpack-scan-team/issues/792

Also on P2: p1HpG7-qtT-p2

**Comment by davemart-in** [comment]
If a link points to a site within our trusted network (Calypso, wp-admin, Jetpack, A4A) we should not be showing these icons.

**Comment by keoshi** [comment]
@davemart-in Would you mind sharing the reasoning behind that opinion?

**Comment by davemart-in** [comment]
We did a full audit of external links and came up with these rules as a result of our observations:

- The external link icon signifies a link will open a new tab. external link
- We should consider all links within our system (Calypso, wp-admin, Jetpack) internal links.
- All links outside of our system should open in a new tab. (YouTube, Fiverr, etc.)
- If linking to a support page, we should open the Help Center; no external link icon is necessary.
- When in doubt, if Automattic owns the property, open the link in _self; no external link icon is necessary.

Here are a few links with additional context:

- pe7F0s-23W-p2
- pekYwv-4Jd-p2

**Comment by keoshi** [comment]
Thanks @davemart-in, will add a comment of the latest post.

**Comment by keoshi** [comment]
Left a comment here: pekYwv-4Jd-p2#comment-4304

**Comment by Initsogar** [comment]
The external icon for these links is intended to warn users that the link will take them to a different external website or domain. Opening new links in a new window or tab is generally considered a bad practice because it can confuse users, disrupt their navigation flow, and lead to accessibility issues ([NNG Group article on new windows and tabs](https://www.nngroup.com/articles/new-browser-windows-and-tabs/)). Additionally, [this behavior cannot be controlled in WordPress Core](https://developer.wordpress.org/reference/functions/add_submenu_page/). Thanks, @keoshi, for the clarifications!

I will proceed with closing this issue as ‘Won’t Fix.’ However, we can continue the discussion in this thread (pekYwv-4Jd-p2#comment-4304) if necessary.

**Slack thread related**: p1734557614283089-slack-C0D96691V
